File: backend/QuizApp/Models/QuizUser.cs
namespace QuizApp.Models;

public class QuizUser
{
    private string Id { get; set; }
    private string Identifier { get; set; }
    private string DeviceId { get; set; }

    public QuizUser(string id, string identifier, string deviceId)
    {
        Id = id;
        Identifier = identifier;
        DeviceId = deviceId;
    }
}

File: src/models/QuizUser.ts
export default interface QuizUser {
    id: string;
    identifier: string;
    deviceId: string;
}

export const quizUsersAreEqual = (a: QuizUser, b: QuizUser): boolean => {
    return a.id === b.id
        && a.identifier === b.identifier
        && a.deviceId === b.deviceId
}


